docs/sources/next/javascript-api/_index.md

```diff
@@ -11,6 +11,14 @@ The list of k6 modules natively supported in your k6 scripts.
 
 {{< docs/shared source="k6" lookup="javascript-api/init-context.md" version="<K6_VERSION>" >}}
 
+## Import.meta
```

Comment:
```suggestion
## import.meta
```

Should this be lower-cased? 🤔 
And... can we import the table this from `javascript-api/import.meta/_index.md so we dedup?